src/store.js

```diff
@@ -228,10 +259,18 @@ Store.prototype.getSelected = function() {
 };
 
 /**
+ * Returns selected points in the currently selected feature.
+ * @return {Array<Object>} Selected points.
+ */
+Store.prototype.getSelectedPoints = function() {
+  return this._selectedPoints;
```

Comment:
This is a mutable interface. While we don't currently have anything that depends on these values, we should still protect this data. To do this, we can hold the 'raw data' in store and then convert it to features in the api.js function.

I think we'd want to store to hold an object that looks like this.

```js
[{feature_id: '123', coord_path: '1.1.1', coordinates: [0,0]}}
```

@ChrisBellew does this seem reasonable? Is there something I'm over looking here?